app/models/work.rb

```diff
@@ -0,0 +1,23 @@
+class Work < ApplicationRecord
+  validates :creator, presence: true
+  validates :title, presence: true
+  has_many :votes
```

Comment:
There are validations that you can put here that prevent duplicate users voting on the same work. This is the better way to handle duplicate votes.